Makefile

```diff
@@ -6,9 +6,10 @@ DB_URL_TEST ?= postgres://postgres:postgres@localhost:5434/url_shortener_test?ss
 ############################### Requirements ###############################
 setup:
 	go install github.com/swaggo/swag/cmd/swag@latest
-	go install github.com/golang-migrate/migrate/v4
```

Comment:
Instead of removing, try this:
`go install github.com/golang-migrate/migrate/v4@latest`

If it works is better then using `brew` since will require an extra step

Reply:
weird it works here.
But make sense, Ill merge it